Source/Dafny/DafnyOptions.cs

```diff
@@ -71,18 +71,7 @@ public enum PrintModes {
     public List<string> DafnyPrintExportedViews = new List<string>();
     public bool Compile = true;
 
-    [Flags]
-    public enum CompilationTarget {
-      Csharp = 1,
-      JavaScript = 2,
-      Go = 4,
-      Java = 8,
-      Cpp = 16,
-      Php = 32,
-      Python = 64
-    }
-
-    public CompilationTarget CompileTarget = CompilationTarget.Csharp;
+    public CompilerFactory CompilerFactoryInstance;
```

Comment:
With flags it was obvious we could run multiple compilers at the same time.
How are you going to do it with this? I suggest you use a comma-separated list of compilers

Reply:
> With flags it was obvious we could run multiple compilers at the same time.

I don't think we ever could actually run multiple compilers at the same time, could we?